src/virtual-wan/azext_vwan/_params.py

```diff
@@ -212,6 +212,9 @@ def load_arguments(self, _):
 
     with self.argument_context('network p2s-vpn-gateway connection list') as c:
         c.argument('resource_name', p2s_vpn_gateway_name_type, id_part=None)
+
+    with self.argument_context('network p2s-vpn-gateway vpn-client') as c:
+        c.argument('authentication_method', arg_type=get_enum_type(AuthenticationMethod))
```

Comment:
can you use '--auth-method' for short?

Reply:
I followed the style `az network vnet-gateway vpn-client generate -h`. It used `--authentication-method`.
But I am okay with it. we could change it to `--auth-method`